src/RedisSortedSet.php

```diff
@@ -42,9 +42,14 @@ public function add(array $data): Promise
      *
      * @return Promise
      */
-    public function getRange(int $start, int $end): Promise
+    public function getRange(int $start, int $end, RangeOptions $options = null): Promise
```

Comment:
Why is it `RangeOptions` instead of `RangeByScoreOptions` here?

Reply:
Actually a valid questions. I guess i was mislead by the interface of the redis extension by which I modeled those options while the latest redis docs show a different definition where ZRANGE has even more options:
https://redis.io/commands/zrange/ vs https://redis.io/commands/zrangebyscore/

> Starting with Redis 6.2.0, this command can replace the following commands: [ZREVRANGE](https://redis.io/commands/zrevrange), [ZRANGEBYSCORE](https://redis.io/commands/zrangebyscore), [ZREVRANGEBYSCORE](https://redis.io/commands/zrevrangebyscore), [ZRANGEBYLEX](https://redis.io/commands/zrangebylex) and [ZREVRANGEBYLEX](https://redis.io/commands/zrevrangebylex).

Do you have any opinion on how to deal with this protocol evolution? I can adjust the MR then.

Reply:
I'd probably go with RangeByScoreOptions here for now, as the other options can be used with the specialized commands?

Reply:
I pushed another commit. WDYT?